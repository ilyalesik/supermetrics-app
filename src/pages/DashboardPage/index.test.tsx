import renderer from "react-test-renderer";
import React from "react";
import { DashboardPage } from "./index";

it("renders DashboardPage", () => {
  const tree = renderer
    .create(
      <DashboardPage
        senders={<div>senders</div>}
        posts={<div>posts</div>}
        navigation={<div>navigation</div>}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
