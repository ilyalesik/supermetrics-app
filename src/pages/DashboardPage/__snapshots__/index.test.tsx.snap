// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DashboardPage 1`] = `
<div
  className="dashboard-page"
>
  <div />
  <div>
    <div>
      navigation
    </div>
  </div>
  <div
    className="dashboard-page-senders"
  >
    <div>
      senders
    </div>
  </div>
  <div
    className="dashboard-page-posts"
  >
    <div>
      posts
    </div>
  </div>
</div>
`;
